testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeTablePropertiesCompatibility.java

```diff
@@ -0,0 +1,82 @@
+        extends BaseTestDeltaLakeS3Storage
+{
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testCreateTableWithTablePropertiesOnDelta()
+    {
+        String tableName = "test_dl_create_table_properties_" + randomNameSuffix();
+        String tableDirectory = "databricks-compatibility-test-" + tableName;
+
+        onDelta().executeQuery(format("CREATE TABLE default.%s (col INT) USING DELTA LOCATION 's3://%s/%s' TBLPROPERTIES ('foo'='bar', 'Foo'='Baz')",
+                tableName,
+                bucketName,
+                tableDirectory));
+
+        try {
+            assertEquals("bar", getTablePropertyOnTrino("default", tableName, "foo"));
+            assertEquals("Baz", getTablePropertyOnDelta("default", tableName, "Foo"));
+            assertEquals("bar", getTablePropertyOnDelta("default", tableName, "foo"));
+            assertEquals("Baz", getTablePropertyOnTrino("default", tableName, "Foo"));
+            assertEquals(getTablePropertyOnDelta("default", tableName, "delta.minReaderVersion"), getTablePropertyOnTrino("default", tableName, "delta.minReaderVersion"));
+            assertEquals(getTablePropertyOnDelta("default", tableName, "delta.minWriterVersion"), getTablePropertyOnTrino("default", tableName, "delta.minWriterVersion"));
+        }
+        finally {
+            dropDeltaTableWithRetry("default." + tableName);
+        }
+    }
+
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testCreateTableWithTableFeaturesOnDelta()
+    {
+        String tableName = "test_dl_create_table_features_" + randomNameSuffix();
+        String tableDirectory = "databricks-compatibility-test-" + tableName;
+
+        onDelta().executeQuery(format("CREATE TABLE default.%s (col INT) USING DELTA LOCATION 's3://%s/%s' TBLPROPERTIES ('delta.minReaderVersion'='3', 'delta.minWriterVersion'='7', 'delta.columnMapping.mode'='id')",
+                tableName,
+                bucketName,
+                tableDirectory));
+
+        try {
+            assertEquals(getTablePropertyOnDelta("default", tableName, "delta.columnMapping.mode"), getTablePropertyOnTrino("default", tableName, "delta.columnMapping.mode"));
+            assertEquals(getTablePropertyOnDelta("default", tableName, "delta.feature.columnMapping"), getTablePropertyOnTrino("default", tableName, "delta.feature.columnMapping"));
+            assertEquals(getTablePropertyOnDelta("default", tableName, "delta.minReaderVersion"), getTablePropertyOnTrino("default", tableName, "delta.minReaderVersion"));
+            assertEquals(getTablePropertyOnDelta("default", tableName, "delta.minWriterVersion"), getTablePropertyOnTrino("default", tableName, "delta.minWriterVersion"));
```

Comment:
Can we verify if the number of table properties are same between Trino and Delta?